src/components/Cluster/ClusterDetail/InstanceTypeSelector/InstanceTypeSelector.tsx

```diff
@@ -65,27 +79,41 @@ const InstanceTypeSelector: FC<IInstanceTypeSelector> = ({
         >
           <ListToggler
             role='button'
+            id='machine-type-selector__toggler'
+            aria-expanded={!collapsed}
+            aria-labelledby='available-machines-label'
+            tabIndex={0}
             onClick={() => setCollapsed(!collapsed)}
             collapsible={true}
+            onKeyDown={handleTabSelect}
             title={`Show/hide available ${plural}`}
           >
-            <i className={`fa fa-caret-${collapsed ? 'right' : 'bottom'}`} />
-            Available {plural}
+            <i
+              className={`fa fa-caret-${collapsed ? 'right' : 'bottom'}`}
+              aria-hidden='true'
+              aria-label='Toggle'
+              role='presentation'
+            />
+            <span id='available-machines-label'>Available {plural}</span>
```

Comment:
Cheeky little improvement to make keyboard navigation easier. This was already in the release version selector.